src/gui/accelerators.c

```diff
@@ -472,6 +505,85 @@ static gboolean bauhaus_slider_reset_callback(GtkAccelGroup *accel_group, GObjec
   return TRUE;
 }
 
+static gboolean bauhaus_combobox_next_callback(GtkAccelGroup *accel_group, GObject *acceleratable,
+                                                 guint keyval, GdkModifierType modifier, gpointer data)
+{
+  GtkWidget *combobox = GTK_WIDGET(data);
+
+  const int currentval = dt_bauhaus_combobox_get(combobox);
+  const int nextval = currentval + 1 >= dt_bauhaus_combobox_length(combobox) ? 0 : currentval + 1;
+  dt_bauhaus_combobox_set(combobox, nextval);
+
+  dt_bauhaus_widget_t *w = (dt_bauhaus_widget_t *)DT_BAUHAUS_WIDGET(combobox);
+
+  if(!gtk_widget_is_visible(GTK_WIDGET(w)) && *w->label)
+  {
+    if(w->module && w->module->multi_name[0] != '\0')
+      dt_control_log(_("%s %s/%s: %s"), w->module->name(), w->module->multi_name, w->label, dt_bauhaus_combobox_get_text(combobox));
+    else if(w->module)
+      dt_control_log(_("%s/%s: %s"), w->module->name(), w->label, dt_bauhaus_combobox_get_text(combobox));
+    else
+      dt_control_log(_("%s"), dt_bauhaus_combobox_get_text(combobox));
+  }
+
+  g_signal_emit_by_name(G_OBJECT(combobox), "value-changed");
+  return TRUE;
+}
+
+static gboolean bauhaus_combobox_prev_callback(GtkAccelGroup *accel_group, GObject *acceleratable,
+                                                 guint keyval, GdkModifierType modifier, gpointer data)
+{
+  GtkWidget *combobox = GTK_WIDGET(data);
+
+  const int currentval = dt_bauhaus_combobox_get(combobox);
+  const int prevval = currentval - 1 < 0 ? dt_bauhaus_combobox_length(combobox) : currentval - 1;
+
```

Comment:
Just a stylistic note - in `bauhaus_combobox_prev_callback` you have empty line before this call. I'm no expert on code style but should be consistent ;)

Reply:
New commit or just amend the last one?

Reply:
amend. or have it a part of another commit in the future. whitespace changes are absolutelly miniscule things.

Reply:
Ok will do. I do want to add the lens correction combobox because I use this one quite a lot, so I'll do it with that commit.